### PR TITLE
Fixed OCL::Logging cleanup and TLSF memory pool destruction in deployer-corba

### PR DESCRIPTION
**Pull request by meyerj**
...and some general syncs between deployer.cpp, deployer-corba.cpp and cdeployer.cpp.

Usage of white-space is quite inconsistent for those files. I only compared them with each other and synced differences for the exact same line of code in order to simplify future comparisons, but tried to update as few lines as possible within a single file. Best viewed with [?w=1](https://github.com/snrkiwi/ocl/pull/2/files?w=1) in GitHub to hide pure white-space changes.

### CODE REVIEW

File: bin/cdeployer.cpp [hunk 1]
```diff
@@ -160,84 +160,110 @@ int main(int argc, char** argv)
...
 	}
+
 #ifdef  ORO_BUILD_LOGGING
     log4cpp::HierarchyMaintainer::getDefaultMaintainer().shutdown();
     log4cpp::HierarchyMaintainer::getDefaultMaintainer().deleteAllCategories();
 #endif
 
 #ifdef  ORO_BUILD_RTALLOC
-    if (!rtMem)
+    if (0 != rtMem)
```
Comment (meyerj) [inline bin/cdeployer.cpp:261]: This was the actual bug.

File: bin/deployer-corba.cpp [hunk 2]
```diff
@@ -245,13 +263,18 @@ int main(int argc, char** argv)
...
 	}
 
+#ifdef  ORO_BUILD_LOGGING
+    log4cpp::HierarchyMaintainer::getDefaultMaintainer().shutdown();
+    log4cpp::HierarchyMaintainer::getDefaultMaintainer().deleteAllCategories();
+#endif
+
 #ifdef  ORO_BUILD_RTALLOC
-    if (!rtMem)
+    if (0 != rtMem)
```
Comment (meyerj) [inline bin/deployer-corba.cpp:272]: Same here.

File: bin/deployer-corba.cpp [hunk 3]
```diff
@@ -245,13 +263,18 @@ int main(int argc, char** argv)
         rc = -1;
 	}
 
+#ifdef  ORO_BUILD_LOGGING
+    log4cpp::HierarchyMaintainer::getDefaultMaintainer().shutdown();
+    log4cpp::HierarchyMaintainer::getDefaultMaintainer().deleteAllCategories();
+#endif
```
Comment (meyerj) [inline bin/deployer-corba.cpp:269]: This section was missing in deployer-corba.cpp compared to deployer.cpp.

File: bin/deployer.cpp [hunk 4]
```diff
@@ -238,13 +239,6 @@ int main(int argc, char** argv)
 #ifdef  ORO_BUILD_RTALLOC
     if (0 != rtMem)
     {
-        std::cout << "TLSF bytes allocated=" << memSize
-                  << " overhead=" << (memSize - freeMem)
-                  << " max-used=" << get_max_size(rtMem)
-                  << " currently-used=" << get_used_size(rtMem)
-                  << " still-allocated=" << (get_used_size(rtMem) - (memSize - freeMem))
-                  << "\n";
-
```
Comment (meyerj) [inline bin/deployer.cpp:247]: I removed those because the same output has already been produces earlier using the RTT Logger.

**Comment by meyerj** [comment]
The commit that fixed the bug in deployer.cpp but not in deployer-corba.cpp was https://github.com/snrkiwi/ocl/commit/12e97bea8cdfc229a839ad5cdf5afe73e00c8aca.

**Comment by snrkiwi** [comment]
Merged to rdt-pu

**Pull request closed without merging**